backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.adoc

```diff
@@ -0,0 +1,37 @@
+:_content-type: ASSEMBLY
+[id="oadp-rosa-backing-up-applications"]
+= Backing up applications ROSA clusters using OADP
+include::_attributes/common-attributes.adoc[]
+:context: oadp-rosa-backing-up-applications
+
+toc::[]
+
+You can use {oadp-first} with {product-rosa} (ROSA) clusters to back up and restore application data.
+
+ROSA is a fully-managed, turnkey application platform that allows you to deliver value to your customers by building and deploying applications.
+
+ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to speed up the building and delivering of differentiating experiences to your customers.
+
+You subscribe to the service directly from your AWS account.
+
+After you create your clusters, you can operate your clusters with the OpenShift web console or through link:https://docs.openshift.com/dedicated/ocm/ocm-overview.html[{cluster-manager-first}]. You can also use ROSA service with OpenShift APIs and command-line interface (CLI) tools.
+
+For additional information about ROSA installation, see link:https://www.redhat.com/en/products/interactive-walkthrough/install-rosa[Installing Red Hat Openshift Service on AWS (ROSA) interactive walkthrough].
+
+Before installing {oadp-first}, you must set up role and policy credentials for OADP so that it can use the {aws-full} API.
+
+This is a two stage process:
+
+. Prepare AWS credentials
+. Install the OADP Operator and give it an IAM role
+
+include::modules/preparing-aws-credentials-for-oadp.adoc[leveloffset=+1]
+
+include::modules/installing-oadp-rosa-sts.adoc[leveloffset=+1]
+
+[id="oadp-rosa-backing-up-and-cleaning"]
+== Example of backing up and cleaning OADP ROSA STS
```

Comment:
```suggestion
== Example: Backing up workload on OADP ROSA STS, with optional cleanup.
```

I feel this might have clearer messaging which tell cu that cleaning up is optional. Perhaps when this merge, we can sync the changes here back to mobb docs.